File: 2nd Sem/C++/Programs/prime.cpp
#include <iostream>
#include <cmath>
using namespace std;

bool isPrime(int x);

int main()
{

    int i, n;

    cout << "Enter a number: " << endl;
    cin >> n;

    if (isPrime(n))
    {
        cout << "Prime" << endl;
    }
    else{
        cout << "Not Prime" << endl;
    }
}

bool isPrime(int x)
{
    int i;
    if (x < 2)
        return false;
    for (int i = 2; i <= sqrt(x); i++)
    {
        if ((x % i) == 0)
            return false;
    }
    return true;
}

File: 2nd Sem/Web/php/prime.php
<?php  
//Write a function to check a number is prime or not.
    function IsPrime($n)  
    {  
     for($x=2; $x<$n; $x++)  
       {  
          if($n %$x ==0)  
              {  
               return 0;  
              }  
        }  
      return 1;  
       }  
    $a = IsPrime(3);  
    if ($a==0)  
    echo 'This is not a Prime Number.....';  
    else  
    echo 'This is  a Prime Number..';  
    ?>   